tests/check.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/bash
+set -euo pipefail
+dn=$(dirname $0)
+srcdir=$(cd ${dn}/.. && pwd)/src
+
+# Verify syntax for sources
+for f in $(echo ${srcdir}/*); do
+    if ! [ -f "$f" ]; then continue; fi
+    shebang=$(head -1 $f)
+    if [[ $shebang =~ ^#!/usr/bin/python.* ]]; then
+        python3 -m py_compile "${f}"
+        echo "OK ${f}"
+        continue
+    fi
+    if [[ $shebang =~ ^#!/.*/bin/.*bash.* ]]; then
```

Comment:
Hmm, that won't match `#!/bin/bash`, right?

Similarly here, just `^#!/.*/bash.*`?